tests/templates/kuttl/resources/00-install-zk.yaml.j2

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: zookeeper.stackable.tech/v1alpha1
+kind: ZookeeperCluster
+metadata:
+  name: druid-zk
+spec:
+  servers:
+    roleGroups:
+      default:
+        replicas: 1
+  version: {{ test_scenario['values']['zookeeper'] }}
```

Comment:
I don't see a benefit testing against all zk versions. I suggest to use (or introduce) zookeeper-latest

Reply:
yeah thought about that, will do.

Reply:
:+1: